lib/utils/documentdb.ts

```diff
@@ -416,7 +458,7 @@ export function mapResultIterator<A, B>(
                   } else {
                     resolve(
                       right<DocumentDb.QueryError, Option<ReadonlyArray<B>>>(
-                        none
+                        some([])
```

Comment:
this is because executeNext() may return empty values even if the iteration has not ended